loadgen/generic_executor.go

```diff
@@ -25,6 +26,7 @@ type genericRun struct {
 	info     ScenarioInfo
 	config   RunConfiguration
 	logger   *zap.SugaredLogger
+	limiter  *rate.Limiter
```

Comment:
Why is this repeated here when it's stored on `config`? In fact, it seems this field is read but never written - I'm not sure anything's actually happening yet.

Reply:
🤦 You're totally right. This is a leftover from my initial draft hardcoded implementation.

I've fixed it, now the construction like this works as expected:
```golang
loadgen.MustRegisterScenario(loadgen.Scenario{
	Description: "Each iteration executes a single workflow with a noop activity.",
	Executor: loadgen.KitchenSinkExecutor{
		DefaultConfiguration: loadgen.RunConfiguration{
			Limiter: rate.NewLimiter(rate.Every(time.Second * 5), 1),
		},
		TestInput: &kitchensink.TestInput{
			WorkflowInput: &kitchensink.WorkflowInput{
				InitialActions: []*kitchensink.ActionSet{
					kitchensink.NoOpSingleActivityActionSet(),
				},
			},
		},
	},
})
```